tagstudio/src/qt/ts_qt.py

```diff
@@ -652,7 +652,11 @@ def clear_select_action_callback(self):
 
     def show_tag_database(self):
         self.modal = PanelModal(
-            TagDatabasePanel(self.lib), "Library Tags", "Library Tags", has_save=False
+            done_callback=lambda: self.preview_panel.update_widgets(),
```

Comment:
```suggestion
            done_callback=self.preview_panel.update_widgets,
```
Minor nit, move between window_title and has_save to match parameters